Draft-Accepted/RFC0066-PowerShell-User-Content-Location.md

```diff
@@ -0,0 +1,133 @@
+---
+RFC: RFC0066
+Author: Justin Chung
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Core
+Comments Due: 07/31/2025
+Plan to implement: Yes
+---
+
+# PowerShell User Content Location
+
+This RFC proposes moving the current PowerShell user content location out of OneDrive &
+default to the `AppData` directory on Windows machines.
+
+## Motivation
+
+```
+As a user,
+I can customize the location where PowerShell user content is installed,
+so that I can avoid problems created by file sync solutions like OneDrive.
+
+As an Admin,
+I must be able to customize the location set for my users either pre/during/post install,
+so that I can better manage my environment and better secure and support my users,
+to reduce unnecessary support tickets to my IT Team.
+```
+
+- PowerShell currently places profile, modules, and configuration files in the user's Documents
+  folder, which is against established conventions for shell configurations and tools.
+- PowerShell content files in OneDrive can lead to unwanted syncing of module files, leading to
+  various issues.
+- There is strong community demand for changing this behavior as the current setup is problematic
+  for many users.
+- Changing the default location would align PowerShell with other developer tools and improve
+  usability.
+
+## Specification
+
+- This will be an experimental feature.
+- The content folder location change will only apply to PowerShell on Windows.
+- Configurability of the content folder will apply to all platforms.
+- A configuration file in the PowerShell user content folder will determine the location of the user
+  scoped **PSModulePath**.
+  - By default, the PowerShell user content folder will be located in the
+    `$env:LOCALAPPDATA\PowerShell`.
```

Comment:
This needs to be in the Microsoft Folder, on the off chance an organisation wants to fork PowerShell and use their own Org Location alongside use of Microsoft Releases.
```suggestion
  - By default, the PowerShell user content folder will be located in the
    `$env:LOCALAPPDATA\Microsoft\PowerShell`.
```

Reply:
I would argue it should stay there to maintain existing Linux path consistency. If an org wants to use their own then they can specify that nested path at their leisure.